_layouts/default.html

```diff
@@ -84,6 +84,9 @@ <h2>
                 </a>
             </h2>
         </div>
+        <div class="pair twitter">
```

Comment:
I like this, but not everyone will have a Twitter profile to show. Can you wrap this in an If block?

Docs on conditional statements: https://help.shopify.com/en/themes/liquid/tags/control-flow-tags

Docs on understanding “truthy” values that can be used in conditional statements: https://shopify.github.io/liquid/basics/truthy-and-falsy/

Reply:
Ah! didn't realize this part of it was part of the PR. Yes, I was just looking up ways to do that. I forgot that additional commits after the PR is opened are basically part of the same PR.

Reply:
Ok, I committed a few changes that I _think_ should address this. It appeared to work locally, with my page showing twitter, yours not -- and no errors in the browser or weird artifacts on the page.